carLot/invEventListeners.js

```diff
@@ -0,0 +1,37 @@
+CarLot = (function (carLot) {
+
+  carLot.activateEvents = function() {
+
+    var searchBar = document.getElementById('search');
+    var car = document.getElementsByClassName('car');
+
+    for(i = 0; i < car.length; i++){
+
+      car[i].addEventListener("click", function() {
```

Comment:
Also, on click of the car element, `clear the value of the text input` in the navbar, and put the cursor in the text input.